packages/slider/src/Slider.js

```diff
@@ -80,7 +68,6 @@ export default class Slider extends Component {
   };
 
   static defaultProps = {
-    defaultValue: "",
     id: generateId("slider"),
```

Comment:
I don't think we should be generating an `id` by default, and probably want to pass `id` along with `...otherProps` unless we are doing some with the `id`.